### ISSUE TITLE
API: accepting arrays

### ISSUE THREAD

**Issue by ln-komandur** [Rello/analytics#55]
Hi, I am trying to use the Rest API to send a JSON array using the Python 3 code snippet below
```
payload = []
count = 0
while count < 9:
    now = datetime.now().strftime('%M:%S')
    payload.append({'dimension1': 'X axis', 'dimension2': now, 'dimension3': 1})
    payload.append({'dimension1': 'Y axis', 'dimension2': now, 'dimension3': 10})
    count = count + 2
    time.sleep(1)
print("Posting", json.dumps(payload)) #  using the dumps method to print and check if JSON is formed correctly
response = requests.post(url, headers=headers, json=payload, auth=(UserID, Password), verify=False)
print("Posted as ",response)
```

The output is as below

> Posting [{"dimension1": "X axis", "dimension2": "21:37", "dimension3": 1}, {"dimension1": "Y axis", "dimension2": "21:37", "dimension3": 10}, {"dimension1": "X axis", "dimension2": "21:38", "dimension3": 1}, {"dimension1": "Y axis", "dimension2": "21:38", "dimension3": 10}, {"dimension1": "X axis", "dimension2": "21:39", "dimension3": 1}, {"dimension1": "Y axis", "dimension2": "21:39", "dimension3": 10}, {"dimension1": "X axis", "dimension2": "21:40", "dimension3": 1}, {"dimension1": "Y axis", "dimension2": "21:40", "dimension3": 10}, {"dimension1": "X axis", "dimension2": "21:41", "dimension3": 1}, {"dimension1": "Y axis", "dimension2": "21:41", "dimension3": 10}]
/usr/lib/python3/dist-packages/urllib3/connectionpool.py:860: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
Posted as  <Response [200]>
>

Though I get `<Response [200]>`, not even a single record is shown on the basic line graph or its table that is at the 'url' on my next cloud server.

Whereas, when I have only one record in the json object (not as an array) and use the requests.post method retaining everything else  to be the same, it works fine. Since it is time consuming to insert 1 record at a time for my application, I want to send an  array.

I am using a self-signed certificate and would like to ignore the SSL warning for now, as it applies to either of the cases above and seems immaterial to my problem 

Can you please let me know if the rest API method accepts a  JSON Array? If yes, where am I going wrong?

**Comment by Rello** [comment]
Hello,
thank you for your request. I think this is valid because I only used single updates until now.
I see the usecase of this requirement and get back to you

**Comment by Rello** [comment]
implementation started for v2.5

**Comment by Rello** [comment]
available in api v2.0
https://github.com/Rello/analytics/wiki/API
